packages/client-web/src/modules/timeline/model/store.ts

```diff
@@ -38,14 +38,9 @@ export const setEntries = (entries: Entry[]): SetTimelineEntriesAction => ({
 });
 
 export const fetchTimeline =
-  () => (dispatch: AppDispatch, getState: GetState) => {
-    const { date } = getState();
-
-    dispatch(setTimelineFetching(true));
-    return timelineService
-      .get(date)
-      .then((entries) => dispatch(setEntries(entries)))
-      .catch(() => dispatch(setTimelineFetching(false)));
```

Comment:
The `timeline` store was tightly coupled to the `date` store because it was getting the `date` state in this thunk. Now the `date` is an input param and the store (models) are composed in the `timeline` view model.